File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// KiprioApiSuite SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class KiprioApiSuiteMakeContext
{
    public static function call(array $ctxmap, ?KiprioApiSuiteContext $basectx): KiprioApiSuiteContext
    {
        return new KiprioApiSuiteContext($ctxmap, $basectx);
    }
}

File: .sdk/tm/rb/utility/make_context.rb
# KiprioApiSuite SDK utility: make_context
require_relative '../core/context'
module KiprioApiSuiteUtilities
  MakeContext = ->(ctxmap, basectx) {
    KiprioApiSuiteContext.new(ctxmap, basectx)
  }
end
